src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--
-    Microsoft ResX Schema
-
+  <!-- 
+    Microsoft ResX Schema 
+    
     Version 2.0
-
-    The primary goals of this format is to allow a simple XML format
-    that is mostly human readable. The generation and parsing of the
-    various data types are done through the TypeConverter classes
+    
+    The primary goals of this format is to allow a simple XML format 
+    that is mostly human readable. The generation and parsing of the 
+    various data types are done through the TypeConverter classes 
     associated with the data types.
-
+    
     Example:
-
+    
```

Comment:
What made these spaces to appear?

Reply:
As I haven't done any manual change I guess that's VS.

Reply:
@Evangelink Not the first time I see this issue without manual changes.

@mavasani Any idea where to report this possible flaky bug?

Reply:
I suspect VS's resgen tool is consistently adding them, and editors configured to trim trailing whitespace remove it when the file is manually edited. 
I tend to "let VS win" in whitespace conflicts when that is the easiest thing to maintain.

Filing a suggestion bug against resgen to stop emitting the trailing whitespace in the first place would be Goodness.